lib/intacct_ruby/function.rb

```diff
@@ -14,6 +14,7 @@ class Function
       update
       delete
       getAPISession
+      create_potransaction
```

Comment:
totally NB: I don't want to be that guy, but why not `po_transaction`?

```suggestion
      create_po_transaction
```

Reply:
@hoodr it's to match the name of the function that the Sage API then calls
```
<create_potransaction>
    <transactiontype>Purchase Requisition</transactiontype>
    <datecreated>
...
```

Reply:
@sam-peach is right, if you look at how Function.rb works, youll see it basically gets turned into an xml tag